### ISSUE TITLE
Failed to create event bus using cloudformation on localstack

### ISSUE THREAD

**Issue by Kevchan85** [localstack/localstack#3612]
# Type of request: This is a ...

[ X] bug report
[ ] feature request

# Detailed description
When defining an event bus in a coudformation template it raises a warning for unexpected resource
...

## Expected behavior
Because create-event-bus is supported we should also be able to do this through cloudformation on localstack

...

## Actual behavior
```
localstack_1  | 2021-02-16T23:23:35:WARNING:localstack.utils.cloudformation.template_deployer: Unexpected resource type Events::EventBus when resolving references of resource localEventBus: {'Type': 'AWS::Events::EventBus', 'Condition': 'isLocal', 'LogicalResourceId': 'localEventBus', 'Properties': {'Name': 'my-local'}, 'PhysicalResourceId': None}
```
...

# Steps to reproduce
declare an event bus in a cloudformation template and try to deploy it.

## Command used to start LocalStack
docker run -p 4566:4566 localstack

...

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)
yml
```
Resources:
  localEventBus:
    Type: AWS::Events::EventBus
    Properties:
      Name: my-testing

```
Command
```
awslocal cloudformation create-stack --template-body file://$(pwd)/cloudformation/eventbus.yml --stack-name test-stacks
```
...

**Comment by adrG2** [comment]
Hello, I think I have a similar problem using serverless framework:
```
2021-02-24T22:26:53:WARNING:localstack.utils.cloudformation.template_deployer: Unexpected resource type Events::EventBus when resolving references of resource TestEventBus: {'Type': 'AWS::Events::EventBus', 'LogicalResourceId': 'TestEventBus', 'Properties': {'Name': 'io-events-test-eventBus-local-eu-west-1'}}
2021-02-24T22:26:53:DEBUG:localstack.utils.cloudformation.template_deployer: Handling "Add" for resource "TestEventBus" (1/3) type "AWS::Events::EventBus" in loop iteration 1
2021-02-24T22:26:53:INFO:localstack.utils.cloudformation.template_deployer: Action "create" for resource type Events::EventBus not yet implemented - using fallback mechanism
2021-02-24T22:26:53:DEBUG:localstack.utils.cloudformation.template_deployer: Extract resource attribute: Events::EventBus PhysicalResourceId
2021-02-24T22:26:53:DEBUG:localstack.services.cloudformation.cloudformation_api: Unable to update stack "io-events-test-local": Unable to extract attribute "PhysicalResourceId" from "Events::EventBus" model class <class 'moto.events.models.EventBus'> Traceback (most recent call last):
  File "/opt/code/localstack/localstack/services/cloudformation/cloudformation_api.py", line 340, in update_stack
    deployer.update_stack(new_stack)
  File "/opt/code/localstack/localstack/utils/cloudformation/template_deployer.py", line 1713, in update_stack
    self.apply_changes(self.stack, new_stack, stack_name=self.stack.stack_name)
  File "/opt/code/localstack/localstack/utils/cloudformation/template_deployer.py", line 1918, in apply_changes
    return self.apply_changes_in_loop(changes, old_stack, stack_name)
  File "/opt/code/localstack/localstack/utils/cloudformation/template_deployer.py", line 1954, in apply_changes_in_loop
    self.apply_change(change, stack, new_resources, stack_name=stack_name)
  File "/opt/code/localstack/localstack/utils/cloudformation/template_deployer.py", line 2009, in apply_change
    self.update_resource_details(resource_id, result, stack=old_stack, action=stack_action)
  File "/opt/code/localstack/localstack/utils/cloudformation/template_deployer.py", line 1821, in update_resource_details
    physical_id = physical_id or determine_resource_physical_id(resource_id, stack=stack)
  File "/opt/code/localstack/localstack/utils/cloudformation/template_deployer.py", line 1546, in determine_resource_physical_id
    result = extract_resource_attribute(resource_type, {}, attribute or 'PhysicalResourceId',
  File "/opt/code/localstack/localstack/utils/cloudformation/template_deployer.py", line 763, in extract_resource_attribute
    raise Exception('Unable to extract attribute "%s" from "%s" model class %s' % (
Exception: Unable to extract attribute "PhysicalResourceId" from "Events::EventBus" model class <class 'moto.events.models.EventBus'>

```

**Comment by Kevchan85** [comment]
Yes @adrG2, that was my original issue. The underlying cloud formation service does not implement create for this resource.

**Comment by whummer** [comment]
Thanks for reporting @Kevchan85 @adrG2 . This should be fixed in #3659 - can you please pull the `latest` Docker image and give it another try? Please report here if the problem persists. Thanks!

**Comment by Kevchan85** [comment]
Will try.
